src/app/components/challenge/challengeoverview/challengeoverview.component.spec.ts

```diff
@@ -16,7 +17,7 @@ describe('ChallengeoverviewComponent', () => {
     TestBed.configureTestingModule({
       declarations: [ ChallengeoverviewComponent ],
       providers: [ ChallengeService, ApiService, GlobalService, AuthService, EndpointsService ],
-      imports: [ HttpClientModule ]
+      imports: [ HttpClientModule, RouterTestingModule ]
```

Comment:
I don't see this change relevant to this PR.

Reply:
ok i will drop that change

Reply:
The thing is if you have mentioned that PR is complete that means you are 100% sure that you have reviewed and tested your changes properly. Iterating the PR review multiple times is not a good thing.

Reply:
@Shekharrajak  i checked this change again actually it is required for testing else travis CI will give testing error. So will have to keep it.

Reply:
Thanks for letting me know this point. You can paste the error here (or link of travis error line), so that it can be tracked later.